gitkraken-client/gitkraken-subscriptions.md

```diff
@@ -30,7 +30,7 @@ To purchase a subscription for the first time follow these steps:
 
 You can cancel your subscription at any time. To do so, login to [app.gitkraken.com](https://app.gitkraken.com/), select the organization, select "Subscription" on the left side, and select "<span style='color: yellow;'>Cancel Subscription</span>." Follow the flow to complete cancelation. Once canceled, you will keep your subscription for the remainder of its billing period. Once the period is up, everyone in the organization will lose access to the subscription.
 
-When your subscription is set to cancel but you can still use it, your organization will be labeled as "<span style='color: red;'>canceled</span>." When your subscription is completely canceled and the billing period has passed, it will be lableld as "<span style='color: red;'>expired</span>."
+When your subscription is set to cancel but you can still use it, your organization will be labeled as "<span style='color: red;'>non-renewing</span>." When your subscription is completely canceled and the billing period has passed, it will be lableld as "<span style='color: red;'>non-renewing</span>."

```

Comment:
When your subscription is completely canceled and the billing period has passed, it will be _lableld_ as "<span style='color: red;'>_non-renewing_</span>."
Typo on labeled
Is not expired or canceled?

Reply:
this was incorrect - updated.